scripts/apply-migrations.ts

```diff
@@ -0,0 +1,292 @@
+import { readFileSync, readdirSync } from 'fs'
+import { SQL } from 'bun'
+import { join } from 'path'
+import { loadEnvConfig } from '@next/env'
+
+/**
+ * Load environment variables from .env files
+ * This ensures our database connection string is available
+ */
+loadEnvConfig(process.cwd())
+
+/**
+ * Environment validation
+ * Fail fast if required environment variables are missing
+ */
+if (!process.env.POSTGRES_CONNECTION_STRING) {
+  console.error('❌ POSTGRES_CONNECTION_STRING environment variable is not set')
+  process.exit(1)
+}
+
+// Initialize database connection
+const db = new SQL(process.env.POSTGRES_CONNECTION_STRING)
+
+/**
+ * Creates or verifies the existence of the migrations tracking table
+ * This table keeps track of which migrations have been applied to prevent
+ * duplicate runs and provide an audit trail
+ */
+async function ensureMigrationsTable() {
+  try {
+    await db(`
```

Comment:
this isn't working because correct syntax is 
```suggestion
    await db`
```